src/skin/qml/qmlplayermanagerproxy.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+#include <QObject>
+#include <QString>
+
+class PlayerManager;
+
+namespace mixxx {
+namespace skin {
+namespace qml {
+
+class QmlPlayerManagerProxy : public QObject {
+    Q_OBJECT
+  public:
+    explicit QmlPlayerManagerProxy(PlayerManager* pPlayerManager, QObject* parent = nullptr);
+
+    Q_INVOKABLE QObject* getPlayer(const QString& deck);
+
+  private:
+    const PlayerManager* m_pPlayerManager;
```

Comment:
~~I suggest to use `const QPointer<PlayerManager>` that gets reset implicitly.~~

-> `const std::shared_ptr<PlayerManager>`

Reply:
Do we need a typedef then?